### ISSUE TITLE
Full Accessibility & 508 Office Audit - Veteran Rapid Retraining Assistance Program (VRRAP)

### ISSUE THREAD

**Issue by desireeturner** [department-of-veterans-affairs/va.gov-team#25373]
## Steps to complete the Full Accessibility & 508 Audit: 
- [x] VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit *(Platform understands that other teams may be accountable for fixing the defects identified at Staging Review. These defects are exempt and will not block a team from scheduling this touchpoint)*
- [x] VFS Product Manager: Create this issue and fill in your team and feature name in the title
- [x] VFS: Link to this issue once created in [#vfs-platform-support](https://dsva.slack.com/archives/CBU0KDSB1) in Slack; tag @ Trevor Pierce
- [x] Platform: Create sub-issues attached to this epic with findings. Audits are performed on a first-come-first-serve basis and are scheduled with each biweekly sprint
- [x] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] Platform: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886)
- [x] VFS: Close this ticket when all steps above are completed.

## Artifacts
- Access information for the tool or feature:
    - Staging: https://www.staging.va.gov/education/other-va-education-benefits/veteran-rapid-retraining-assistance/apply-for-vrrap-form-22-1990s/introduction 
    - Production: https://www.va.gov/education/other-va-education-benefits/veteran-rapid-retraining-assistance/apply-for-vrrap-form-22-1990s/introduction
- Link to verification that accessibility checks are complete:
  - VRRAP test plan: https://dsvavsp.testrail.io/index.php?/plans/view/1617
  - The accessibility test specifically is in this test run: https://dsvavsp.testrail.io/index.php?/runs/view/1843
- List of known issues: None

## Additional information
As part of this review, Platform will send your product to VA's 508 office for approval (as applicable).

Please refer to the [Platform Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by shiragoodman** [comment]
hey @desireeturner. we have added this work to our next sprint which begins June 10. We will follow up when the work begins. Thanks!

**Comment by shiragoodman** [comment]
@desireeturner the ticket has been assigned to @noahgelman. Work should begin shortly.

**Comment by dneel-bah** [comment]
> @desireeturner the ticket has been assigned to @noahgelman. Work should begin shortly.

@shiragoodman Has work begun on this review/been completed?

**Comment by noahgelman** [comment]
@shiragoodman  @dneel-bah 

It's done, I attached the feedback to the wrong ticket.

**Comment by shiragoodman** [comment]
thanks @noahgelman !

**Comment by dneel-bah** [comment]
Closing this issue as the attached issues have been fixed / passed to platform.
